### PR TITLE
fix: sidebar highlight for variant path

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksfix-sidebar-highlight-hashicorp.vercel.app/waypoint/tutorials/get-started-nomad-cloud/get-started-nomad-cloud) 🔎
- [Asana task](https://app.asana.com/0/1204333057896641/1204651148003825) 🎟️

## 🗒️ What

Fixes an issue where the tutorial sidebar item wouldn't highlight

## 🧪 Testing

- Visit a variant path [with a query param](https://dev-portal-git-ksfix-sidebar-highlight-hashicorp.vercel.app/waypoint/tutorials/get-started-nomad-cloud/get-started-nomad-cloud?variants=operating-system%3Awindows), validate that the sidebar highlights as expected, [compare with upstream](https://dev-portal-git-ksvariants-assembly-hashicorp.vercel.app/waypoint/tutorials/get-started-nomad-cloud/get-started-nomad-cloud?variants=operating-system%3Awindows)

### CODE REVIEW

File: src/components/sidebar/components/sidebar-nav-menu-item/index.tsx [hunk 1]
```diff
@@ -100,9 +102,28 @@ const SidebarNavLinkItem = ({ item }: SidebarNavLinkItemProps) => {
 
 	// Determine the trailing icon to use, if any
 	const trailingIcon = isExternal ? <IconExternalLink16 /> : item.trailingIcon
+	const [ariaCurrent, setAriaCurrent] = useState<'page' | undefined>()
+
+	if (ariaCurrent) {
+		console.log(ariaCurrent, href, item.isActive, '*** is current')
+	}
```
Comment (kendallstrautman) [inline src/components/sidebar/components/sidebar-nav-menu-item/index.tsx:109]:
```suggestion
```

File: src/components/tutorial-meta/components/variant-list/desktop/desktop-variant-list.module.css [hunk 2]
```diff
@@ -38,7 +38,6 @@
 		background-color: var(--token-color-foreground-action);
 		border-color: var(--token-color-foreground-action-hover);
 		color: var(--token-color-foreground-high-contrast);
-		cursor: not-allowed;
```
Comment (kendallstrautman) [inline src/components/tutorial-meta/components/variant-list/desktop/desktop-variant-list.module.css:41]: Quick lil style fix here https://app.asana.com/0/1204333057896641/1204669563792088

**Review by BrandonRomano — APPROVED** [review]
🙈

**Review by BRKalow — APPROVED** [review]
🤷‍♂️ 🤞 😱

**Pull request merged**